src/matrix.jl

```diff
@@ -226,13 +226,11 @@ end
 @doc raw"""
     eigenvalues(M::MatElem{T}) where T <: RingElem
 
-Return the eigenvalues of `M` which lie in `base_ring(M)`.
+Return the eigenvalues of `M` which lie in the field `base_ring(M)`.
```

Comment:
Why "the field"? The signature allows matrices over non-rings, and the code also look as if it does so?